src/DataReader.php

```diff
@@ -68,7 +68,6 @@ public function __construct(Command $command, $config = [])
     {
         $this->_statement = $command->pdoStatement;
         $this->_statement->setFetchMode(\PDO::FETCH_ASSOC);
-        parent::__construct($config);
```

Comment:
you still have `$config` in the constructor signature.

Btw, can someone point me to the discussion, on why this has been removed from Yii core?

Reply:
I don't know.
Just fix "Error: Cannot call constructor".